src/game/CLASSES.ts

```diff
@@ -30,6 +30,22 @@ export enum CLASSES {
   WARRIOR,
 }
 
+export const CLASS_NAMES: { [classId: number]: { name: string } } = {
```

Comment:
TIL that we didn't have class ids anywhere

Reply:
I was quited perplexed by this, apparently we convert the classes from the string/spec instead.